tests/labhub_test.py

```diff
@@ -104,22 +98,15 @@ def test_create_issue_cmd(self):
             plugins.labhub.LabHub, logging.ERROR, {'BACKEND': 'text'}
         )
         labhub.activate()
-        labhub.REPOS = {'repository': self.mock_repo,
-                        'repository.github.io': self.mock_repo}
+        labhub.REPOS = {'repository': self.mock_repo,'repository.github.io': self.mock_repo}
 
-        testbot_public.assertCommand('!new issue repository this is the title\nbo\ndy',
-                              'Here you go')
+        testbot_public.assertCommand('!new issue repository this is the title\nbo\ndy','Here you go')
 
-        labhub.REPOS['repository'].create_issue.assert_called_once_with(
-            'this is the title', 'bo\ndy\nOpened by @None at [text]()'
-        )
+        labhub.REPOS['repository'].create_issue.assert_called_once_with('this is the title', 'bo\ndy\nOpened by @None at [text]()')
 
-        testbot_public.assertCommand('!new issue repository.github.io another title\nand body',
-                              'Here you go')
+        testbot_public.assertCommand('!new issue repository.github.io another title\nand body', 'Here you go')
 
-        labhub.REPOS['repository.github.io'].create_issue.assert_called_with(
-            'another title', 'and body\nOpened by @None at [text]()'
-        )
+        labhub.REPOS['repository.github.io'].create_issue.assert_called_with('another title', 'and body\nOpened by @None at [text]()')
```

Comment:
Please keep the line length at most 80 characters.

Reply:
Ok!

Reply:
@nvzard , why didnt CI pick this up?  Please fix that.

Reply:
Done https://github.com/coala/corobo/issues/565